src/database/sequelize.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Sequelize = require('sequelize');
+
+module.exports = new Sequelize('postgres', 'postgres', '8888', {
```

Comment:
The password for the PostgreSQL connection should not be hardcoded. Use environment variables instead.